docs/user/pings/deletion_request.md

```diff
@@ -0,0 +1,45 @@
+# The `deletion_request` ping
+
+## Description
+
+This ping is submitted when a user opts out of sending technical and interaction data.
+
+This ping contains the client id. This ping does not contain any environment data.
```

Comment:
```suggestion
This ping contains the client id.
```

Well... `client_info` technically is environment data. I'd remove that .

Reply:
oh :D copy-pasta from desktop.